### PR TITLE
Drop Ruby warnings

### PR DESCRIPTION
**Pull request by franckverrot**
A quick attempt to get rid of the Ruby warnings popping up in dependent gems/apps.

Thanks for maintaining `i18n_data` in any case!

### CODE REVIEW

File: lib/i18n_data.rb [hunk 1]
```diff
@@ -74,9 +74,9 @@ def recognise_code(type, search)
       common_languages = ['EN','ES','FR','DE','ZH']
       langs = (common_languages + (languages.keys - common_languages))
 
-      langs.each do |code|
+      langs.each do |lang|
         begin
-          send(type, code).each do |code, name|
+          send(type, lang).each do |code, name|
```
Comment (grosser) [inline lib/i18n_data.rb]: using code gave a warning ?

**Comment by grosser** [comment]
can you fix the remaining warnigns too ?
```
/home/travis/build/grosser/i18n_data/lib/i18n_data/version.rb:2: warning: already initialized constant Version
/home/travis/build/grosser/i18n_data/lib/i18n_data/version.rb:2: warning: already initialized constant VERSION
/home/travis/build/grosser/i18n_data/spec/i18n_data_spec.rb:28: warning: useless use of == in void context
/home/travis/build/grosser/i18n_data/spec/i18n_data_spec.rb:80: warning: useless use of == in void context
/home/travis/build/grosser/i18n_data/spec/i18n_data_spec.rb:120: warning: useless use of == in void context
/home/travis/build/grosser/i18n_data/spec/i18n_data_spec.rb:160: warning: useless use of == in void context
```

**Comment by franckverrot** [comment]
Sure, I'll update the PR in a few hours.




Thanks!



—

File: lib/i18n_data.rb (hunk 1 above)
Comment (franckverrot) [inline lib/i18n_data.rb]: Yes, it was shadowed by the closure's parameters.

**Comment by franckverrot** [comment]
@grosser It's done :-)

Thanks!

**Comment by grosser** [comment]
not a big fan of the `eq` but having no warnings is kind of nice even though all these warnings were useless ...

**Comment by franckverrot** [comment]
I was hesitating between `should be ==` and `should eq`.

If you prefer the former, I'll send another PR. I have no preferences TBH.

**Comment by grosser** [comment]
0.6.1 is out!

**Comment by grosser** [comment]
nah shorter is nicer :)

**Pull request merged**